src/tools/place_user_tool.rb

```diff
@@ -24,12 +38,23 @@ def onMouseMove(_flags, x, y, view)
     @mouse_input.update_positions(view, x, y)
   end
 
-  def onKeyDown(key, _repeat, _flags, _view)
+  def onKeyDown(key, _repeat, flags, _view)
```

Comment:
why did you remove the underscore here, I think it's still unused, isn't it?

```suggestion
  def onKeyDown(key, _repeat, _flags, _view)
```

Reply:
You are right, I played around with moving the user with ctrl + arrow keys, but that did not work, but forgot to remove the change